WordPress/Classes/System/WordPressAppDelegate.swift

```diff
@@ -210,10 +224,11 @@ class WordPressAppDelegate: UIResponder, UIApplicationDelegate {
         PushNotificationsManager.shared.registerForRemoteNotifications()
 
         // Deferred tasks to speed up app launch
-        DispatchQueue.global(qos: .background).async {
+        DispatchQueue.global(qos: .background).async { [weak self] in
             MediaCoordinator.shared.refreshMediaStatus()
             PostCoordinator.shared.refreshPostStatus()
             MediaFileManager.clearUnusedMediaUploadFiles(onCompletion: nil, onError: nil)
+            self?.uploadsManager.resume()
```

Comment:
Could there be any conflict with the media and posts refresh tasks and the upload manager resume being performed async?

Reply:
If there is I'm not aware of any.  Both `PostCoordinator` and `MediaCoordinator` have their own operation queues, which makes me think they're prepared to be thread safe.

If we find any issue with this we can tackle it incrementally.